### ISSUE TITLE
Version ranges

### ISSUE THREAD

**Issue by elharo** [GoogleCloudPlatform/cloud-opensource-java#1016]
Version ranges can override the usual level order version selection algorithm for multiple versions of a project. Do our tools take this into account?

**Comment by suztomo** [comment]
No, `DependencyGraphBuilder.levelOrder` does not take ranges into account.
